kitchen.ec2.yml

```diff
@@ -22,7 +22,7 @@ driver:
     http_put_response_hop_limit: 1
     instance_metadata_tags: 'enabled'
   iam_profile_name: <%= ENV['KITCHEN_IAM_PROFILE'] %>
-  instance_type: <%= ENV['KITCHEN_INSTANCE_TYPE'] || 't2.micro' %>
+  instance_type: <%= ENV['KITCHEN_INSTANCE_TYPE'] || 'c5n.xlarge' %>
```

Comment:
This unrelated change should be moved to a separate PR and it's also not mentioned in the PR description